File: aspnet-core/src/ERP.Application.Shared/Manufacturing/SetupForms/Dtos/MFWCTOLDto.cs
using System;
using Abp.Application.Services.Dto;

namespace ERP.Manufacturing.SetupForms.Dtos
{
    public class MFWCTOLDto : EntityDto
    {
        public int? DetID { get; set; }

        public string WCID { get; set; }

        public string TOOLTYID { get; set; }

        public string TOOLTYDESC { get; set; }

        public string UOM { get; set; }

        public double? REQQTY { get; set; }

        public double? UNITCOST { get; set; }

        public double? TOTALCOST { get; set; }

    }
}

File: angular/src/app/main/manufacturing/shared/dto/mfwctol.dto.ts
import { StringifyOptions } from "querystring";

export class CreateOrEditMFWCTOLDto {
    srNo:number;
    detID?: number;
    wCID: string;
    tOOLTYID: string;
    tOOLTYDESC: string;
    uOM: string;
    rEQQTY?: number;
    uNITCOST?: number;
    tOTALCOST?: number;
}